### DISCUSSION TITLE
Add Stress testing

### DISCUSSION THREAD

**Discussion by ouuan** [cpeditor/cpeditor#162] [Ideas]
**Is your feature request related to a problem? Please describe.**

I can't use stress testing inside the editor.

**Describe the solution you'd like**

Add a window for stress testing, it has two (or three, including the current solution, maybe sync with the main editor) editors — one for the other solution and one for the test case generator, you can load them from files, maybe load from other tabs. You can set the time limit, and run until finding a diff between the two solutions. The number of correct test cases and used time for each test case can be displayed while stress testing.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by ghost** [comment]
Hi **ouuan**
Thanks for Opening a new Issue here. I will also thank you for strictly following the Issue Templates. 
I will add an appropriate tag to your Issue.  

Have Patience, One of our Cool Contributors will help you soon.

**Comment by rewhile** [comment]
Can you consider this high priority?

It is an important feature I think.

**Comment by ShizuhaAki** [comment]
I think that it can be released as a separate Qt app (cpcompare), and add an option to launch it from the editor.

**Comment by rewhile** [comment]
I don't think another app is a good idea

**Comment by ouuan** [comment]
We need a powerful system, such as generator templates, choose files in the tabs/local files, and add the counter case in the test cases, otherwise, it won't be better than a shell script.

**Comment by ouuan** [comment]
This is moved back to #917.
